js-challenge/weather-api/index.js

```diff
@@ -0,0 +1,56 @@
+
+function areaWhether() {
+    //プルダウンの情報を受け取る
+    var city = document.getElementById("city").value;
+    var text = '';
+    if (city === 'London') {
+        currentWhether();
+    }
+    else if (city === 'Tokyo') {
+        currentWhether();
+    }
+    else if (city === 'Paris') {
+        currentWhether();
+    }
+    else if (city === 'New York') {
+        currentWhether();
+    }
+    else if (city === 'Indonesia') {
+        currentWhether();
+    }
```

Comment:
どの分岐でも呼び出す関数は一緒なのでまとめられますね。

```suggestion
  if (['London', 'Tokyo', 'Paris', 'New York', 'Indonesia'].includes(city)) {
    currentWhether();
  }
```